src/app/pages/AddToken.tsx

```diff
@@ -205,14 +205,14 @@ const Form: React.FC = () => {
           }
         }
 
-        const { symbol = "", name = "", decimals = 0, icon = "" } =
+        const { symbol = "", name = "", decimals = 0, logo = "" } =
           tokenData || {};
 
         setValue([
           { symbol: symbol.substr(0, 5) },
           { name: name.substr(0, 50) },
           { decimals },
-          { iconUrl: icon },
+          { iconUrl: logo },
```

Comment:
Even this change doesn't help to add logo for now :disappointed:

Reply:
No worries, that's ok. We're considering having a separate proxy/image cache for efficient and secure delivery of metadata images. It's planned for the next iterations.